src/docs/release/breaking-changes/eliminating-nullok-parameters.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Eliminating `nullOk` Parameters
+description: To eliminate `nullOk` parameters to help with API sanity in the face of null safety.
+---
+
+## Summary
+
+This migration guide describes conversion of code that uses the `nullOk`
+parameter on multiple `of` static accessors and related accessors to use
+alternate APIs with nullable return values.
+
+## Context
+
+Flutter has a common pattern of allowing lookup of some types of widgets
+(`InheritedWidget`s) using static member functions that are typically called
```

Comment:
nit: maybe make InheritedWidget a link to API docs?

Reply:
Done.